Source/Tenancy/LoggerExtensions.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Dolittle.Runtime.Tenancy
+{
+    /// <summary>
+    /// Represents a extensions for <see cref="ILogger" />.
+    /// </summary>
+    static class LoggerExtensions
+    {
+        static readonly Action<ILogger, Exception> _failure = LoggerMessage
+            .Define(
+                LogLevel.Warning,
+                new EventId(1231141, nameof(Failure)),
+                "An error occurred");
```

Comment:
Seems like we can make a little more specific message for what's happening here.

Reply:
Done